Components/Alert/alert.css

```diff
@@ -0,0 +1,61 @@
+.alert-container
+{
+    width: 100%;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    background-color: #F5F7F9;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 1rem;
+    box-sizing: border-box;
```

Comment:
remove box-sizing border box from here and add it to the body or reset css

Reply:
Done, thanks for this box-sizing advice.